setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='iopipe',
-    version='0.9.1',
+    version='0.9.2',
```

Comment:
Shouldn't this go back to importing constants so that the version is easy to update in one place?

Reply:
@pselle The problem with this approach is that it makes the `setup.py` dependent on the `iopipe` module which itself is dependent on requests/boto. So if the user has neither requests nor boto, they won't be able to install until they resolve either. Since neither are hard dependencies and are dependent on the env (i.e. if the agent is running in a lambda or not.), this seems like the best way to go.